### DISCUSSION TITLE
Multi things missing from the directions.

### DISCUSSION THREAD

**Discussion by chaoscode** [Zero6992/chatGPT-discord-bot#179] [General]
The guide to setup is deceptively simple because its missing very key steps I had to solve.

It would be helpful for a more detailed guide for setup that includes things like the aiohttp requirement and more detailed discord key bot setup.

**Comment by KingBain** [comment]
Make the suggestions to the readme you want to see and then submit the changes for review. Github makes this fairly easy.
